libs/frontend/ui-theme/src/css/components/prosemirror-view.postcss

```diff
@@ -0,0 +1,66 @@
+/** This file is a copy of "node_modules/prosemirror-view/style/prosemirror.css" */
```

Comment:
This css file is coped from `node_modules/prosemirror-view/style/prosemirror.css`. This is necessary because it covers some of the style (i.e. `.ProseMirror pre`) in ProseMirror for code_block. Including these styles also solves the pre-wrap warning from ProseMirror.